### ISSUE TITLE
Import issue 

### ISSUE THREAD

**Issue by loxK** [ahabra/hash-router#1]
Hello,

Since 0.2.1 I have issue. I use it the recommended way :

```js
import * as HashRouter from '@techexp/hash-router'
```

And get this error from webpack :

```js
 Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (9:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| // src/HashRouter.js
> import { Stringer as Stringer2 } from "@techexp/jshelper";
| 
| // src/utils/RouterUtils.js
Details:
    domainEmitter: [object Object]
    domainThrown: false

```

**Comment by ahabra** [comment]
Hello,

I just published version 0.2.3 with a fix. Please try. Sorry for the long delay, but I only get to work on my open source in the weekend.
The cause of your problem (I think) is that I added "type=commonjs" to package.json. That was a mistake :(

Thank you.

**Comment by loxK** [comment]
Thanks you for the fix

https://github.com/ahabra/hash-router/commit/20a859596b5ebf993082532e2eb6d85bd00488f7
